include/nbl/system/ISystem.h

```diff
@@ -35,6 +37,7 @@ class ISystem final : public core::IReferenceCounted
     enum E_REQUEST_TYPE
     {
         ERT_CREATE_FILE,
+        ERT_CREATE_BUFFERED_FILE,
```

Comment:
you dont need a request to thread to create memory file

Reply:
I second this